autopilot/model-explainability/explaining_customer_churn_model.ipynb

```diff
@@ -90,7 +90,7 @@
    "metadata": {},
```

Comment:
<p>It should not be that involved to write a function, call it <code>create_automl_job</code>  that creates an automl job on a simple dataset and return the job id.  So we can do something like</p><p> </p><pre class="ql-syntax" spellcheck="false">automl_job_name = None # replace None by your autopilot job that exists

if automl_job_name == None:
  automl_job_name = create_automl_job()
 
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1967/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Will add this in a bit